static/sass/_snapcraft_p-navigation.scss

```diff
@@ -2,8 +2,10 @@
   $navigation-border-color: rgba(255, 255, 255, .2);
   $navigation-hover-color: rgba(0, 0, 0, .92);
   $navigation-highlight-color: rgba(250, 90, 64, 1);
+  $navigation-background-color: rgba(51, 51, 51, 1);
```

Comment:
Can we set `$nav-bg-color` to `#333` and remove the `background: $navigation-background-color` below?

`#333` is easier to understand than rgba and We should fall back to vanilla where we can. We're overriding a variable with a variable here - seems counter-intuitive.

Reply:
LGTM!